src/services/util.js

```diff
@@ -1,7 +1,22 @@
+import {
+  getLocalStorage,
+  TOKEN_KEY,
+  USER_DATA_KEY,
+  isBrowser,
+} from './localStorage'
 export const getYoutubeThumbnail = url => {
   if (url && url.match(/youtube/)) {
     const id = url.match(/v=(.+)$/)[1]
 
     return `https://i.ytimg.com/vi/${id}/mqdefault.jpg`
   }
 }
+
+export const isLoggedIn = () => {
+  return getLocalStorage(TOKEN_KEY) ? true : false
+}
+
+export const getUser = () =>
+  isBrowser && window.localStorage.getItem(USER_DATA_KEY)
+    ? JSON.parse(window.localStorage.getItem(USER_DATA_KEY))
+    : {}
```

Comment:
good defaulting to `{}` .. maybe you can default to `{username: ''}` to make sure the code that has `getUser().username.toUppercase()` doesn't break

(come to think about it, I think there is no need to upper case it though, we should display what user added)